pyqtorch/primitive.py

```diff
@@ -76,6 +76,14 @@ def __init__(self, target: int):
         super().__init__(OPERATIONS_DICT["SDAGGER"], target)
 
 
+class Proj(Primitive):
```

Comment:
why not `Projector`? otherwise lgtm

Reply:
Indeed.

Reply:
I guess it was for the sake of conciseness but hey.